### ISSUE TITLE
Choosing anything from a game's tool tab returns the main window to the library.

### ISSUE THREAD

**Issue by dimitrisp2** [Heroic-Games-Launcher/HeroicGamesLauncher#850]
**Describe the bug**
Happy new year everyone! Using anything on a game's tools tab returns me to the library.

**Add Logs**

```
INFO: Legendary location: '/usr/bin/legendary'
INFO: 
  Heroic Version: 2.0.0 Roronoa Zoro
  Legendary Version:  0.20.21 Anticitizen One (hotfix #2)
  OS: Pop KERNEL: 5.15.8-76051508-generic ARCH: x64
  CPU: AMD Ryzen 7 2700 Eight-Core Processor @3.2 GOVERNOR: schedutil
  RAM: Total: 33.7 GB Available: 29.9 GB
  GRAPHICS: GPU0: NVIDIA GeForce GTX 1660 Ti VRAM: 5941MB DRIVER: 470.86 
  PROTOCOL: x11
  
ERROR: Failed to register protocol with OS.
[14727:0101/011207.783028:ERROR:browser_main_loop.cc(269)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[14727:0101/011207.783120:ERROR:browser_main_loop.cc(269)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
[14727:0101/011207.783220:ERROR:browser_main_loop.cc(269)] Gtk: gtk_widget_add_accelerator: assertion 'GTK_IS_ACCEL_GROUP (accel_group)' failed
WARNING: python command not found
INFO: Command 'python3' found. Version: '3.9.7'
INFO: Frontend: Refreshing Library
INFO: Refreshing library...
INFO: Legendary: Refreshing Epic Games...
INFO: Refreshing Epic Games...
ERROR: Error: spawn '/usr/bin/legendary' ENOENT
INFO: Updating game list
INFO: Game List Updated
INFO: Checking for game updates
INFO: Found 0 game(s) to update
INFO: Getting More details with Legendary info command... 
INFO: checking for heroic updates
```

**To Reproduce**

1. Go to library
2. Enter a game
3. Click on 'Tools'
4. Click on any option (for example Last log or Check compatibility)
5. Main window returns to the library.

**Expected behavior**
Heroic should stay on the game page instead of going back to the library.

**Screenshots**
Screenshots won't help, but see the video below:
https://user-images.githubusercontent.com/3811387/147840975-10b07dc0-bc4a-45ca-9986-91588b09b2be.mp4

**Desktop (please complete the following information):**

- OS: Pop!_OS 21.10
- Heroic Version 2.0.0
- Game: N/A
- Wine/Proton version: N/A
- Hardware specs (RAM, CPU, GPU): 32GB RAM, AMD Ryzen 7 2700 CPU, NVidia Geforce GTX1660 Ti

**Comment by flavioislima** [comment]
This was fixed in the code already, will he fixed on next release. @arielj pinging you as well since you opened a similar issue.

**Comment by arielj** [comment]
@flavioislima is this fix in the main branch? I'm using the latest commit of the main branch and the issue is still present

**Comment by flavioislima** [comment]
Ah, reading the issue again, this is on the Tools menu not on the Tools component we have that is for Wine so that's why I got confused. Because we had the issue there and I fixed it, but this other one didn't happen to me, at least clicking on the latest log works fine.
But, in this case, we will need to use buttons and remove the `href=#` from it, this is how I solved the issue on the wine page.
